### PR TITLE
Update site to gluon v2018.2

### PR DESCRIPTION
**Pull request by krombel**
- [x] update CHANGELOG.md to v2018.1.1
- [x] updated translations (de, en)
- [x] added french translation (just copied. Please review)
- [x] update site.mk
  - gluon now provides a GLUON_FEATURES where all packages get
    prepended with "gluon-" automatically. This is now used
  - use GLUON_FEATURE web-wizard (see [docs](https://gluon.readthedocs.io/en/v2018.1.1/user/site.html#feature-flags)
- [x] use multidomain config instead of several sites (native from gluon)
- [x] set default mesh type to ibss. Some devices do not support both types to gluon suggests this way
- [x] Add sign key for krombel (autoupdate)
- [ ] Cleanup link to https://ffmuc.krombel.de where I publish prebuild images
- [x] cleanup: use ffmuc gluon-packages once https://github.com/freifunkMUC/gluon-packages/pull/5 is merged

### CODE REVIEW

File: site.mk [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+	web-admin \
+	web-autoupdater \
+	web-network \
+	web-wifi-config \
+	mesh-batman-adv-15 \
+	mesh-vpn-fastd \
+	radvd \
+	status-page \
+	config-mode-domain-select \
+	ebtables-filter-arp-ffmuc \
```
Comment (rotanid) [inline site.mk]: this should be obsoleted by now by gluons own features.

File: site.mk [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+	ebtables-filter-multicast-ffmuc \
```
Comment (rotanid) [inline site.mk]: this should be obsoleted by now by gluons own features.

**Comment by rotanid** [comment]
maybe you also need to set the GLUON_WLAN_MESH in site.mk.
on the freifunk forum, there's a thread with migration tips for each gluon release

File: domains/ffmuc25.conf [hunk 3]
```diff
@@ -112,49 +107,6 @@
         },
       },
     },
-    bandwidth_limit = {
-      enabled = false,
-      egress = 1500,
-      ingress = 12000,
-    },
-  },
-
```
Comment (saerdnaer) [inline domains/ffmuc25.conf]: Ist es Absicht das die Autoupdate-Funktion deaktivert wird?

Comment (krombel) [reply domains/ffmuc25.conf]: Jein. Hiermit wird die Funktion nicht deaktiviert sondern dedupliziert.
Die auto-update config ist in der site.conf und wird für die einzelnen Domains übernommen.
https://github.com/freifunkMUC/site-ffm/blob/5a7f4f60c5899798a007aaf15d3e8ad65b144011/site.conf#L56-L91

File: site.mk (hunk 1 above)
Comment (krombel) [reply site.mk]: [This](https://github.com/freifunkMUC/gluon-packages/blob/master/gluon-ebtables-filter-arp-ffmuc/files/lib/gluon/ebtables/109-mcast-drop-non-site) looks like it is specific for ffmuc. Should it be skipped either way?

File: site.mk (hunk 2 above)
Comment (krombel) [reply site.mk]: [This](https://github.com/freifunkMUC/gluon-packages/blob/master/gluon-ebtables-filter-multicast-ffmuc/files/lib/gluon/ebtables/110-mcast-allow-cjdns) makes me think, that this allows multicast for cjdns. Is that really supported in gluon out of the box?
Either way: The question stays: Should we keep (cjdns multicast support) of remove this entirely?

File: domains/ffmuc25.conf (hunk 3 above)
Comment (rotanid) [reply domains/ffmuc25.conf]: ich würde dazu raten, aber dann einen key von dir (und vielleicht einem weiteren der heute aktiven freifunker in münchen) hinzuzufügen.

Comment (krombel) [reply domains/ffmuc25.conf]: Hab ich mal als Kommentar in die PR description geschrieben, damit es nicht untergeht.
Danke für den Hinweis.

File: site.mk (hunk 1 above)
Comment (rotanid) [reply site.mk]: please, read the release notes starting with... v2016.2 i guess.
this package does the same and more, configured via site.conf: https://github.com/freifunk-gluon/gluon/tree/master/package/gluon-ebtables-source-filter
https://github.com/freifunk-gluon/gluon/blob/master/docs/package/gluon-ebtables-source-filter.rst

**Comment by rotanid** [comment]
i think you also want the packages ebtables-source-filter and ebtables-limit-arp (docs can be found in gluons docs/package/ folder)

**Comment by rotanid** [comment]
another thing, you can save ~30% traffic overhead if you switch to MTU 1312 instead of 1280
saving means more speed for cheap devices. we made the switch in august 2017.
reasons are explained in detail here: https://github.com/freifunk-gluon/gluon/blob/master/docs/user/faq.rst
only downside is, that you have to use new fastd ports (again), as fastd can't support more than one mtu per instance.

File: modules [hunk 4]
```diff
@@ -1,5 +1,5 @@
 GLUON_SITE_FEEDS='ffmuc'
 
-PACKAGES_FFMUC_REPO=https://github.com/krombel/gluon-packages.git
+PACKAGES_FFMUC_REPO=https://github.com/freifunkMUC/gluon-packages.git
 PACKAGES_FFMUC_BRANCH=add_domain-migration-ffmuc
```
Comment (rmoriz) [inline modules]: I guess PACKAGES_FFMUC_BRANCH should be "master"?

Comment (krombel) [reply modules]: Ohh yes of course. Thanks for pointing it out!

**Comment by saerdnaer** [comment]
Gibts noch Blocker/TODOs die einem Merge widersprechen?

**Comment by rotanid** [comment]
@saerdnaer definitely #45 (and the blockers for that), maybe more than this

**Comment by krombel** [comment]
As I have now created a dedicated branch for this task I am using this now as source branch.
This PR is therefore superseded by #49

**Pull request closed without merging**